### PR TITLE
Moved kvutils to under participant-state-kvutils

### PR DESCRIPTION
**Pull request by miklos-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state-kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/package.scala [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ledger.participant-state-kvutils.src.main.scala.com.daml.ledger.participant.state.kvutils
+
+- state - kvutils.src.main.scala.com.daml.ledger.participant.state.kvutils
+
+package object api {
+
+}
```
Comment (SamirTalwar) [inline ledger/participant-state-kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/package.scala]: Well, this seems quite odd.

File: ledger/api-server-damlonx/reference-v2/BUILD.bazel [hunk 2]
```diff
@@ -41,7 +41,7 @@ da_scala_binary(
         "//ledger/ledger-api-common",
         "//ledger/ledger-api-health",
         "//ledger/participant-state",
-        "//ledger/participant-state/kvutils",
+        "//ledger/participant-state-kvutils:kvutils",
```
Comment (SamirTalwar) [inline ledger/api-server-damlonx/reference-v2/BUILD.bazel]: If you're gonna rename the package, I'd suggest also renaming the targets accordingly.

File: ledger/participant-state-kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/package.scala (hunk 1 above)
Comment (miklos-da) [reply ledger/participant-state-kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/package.scala]: Accidentally left here -- removed.

**Comment by miklos-da** [comment]
> This makes sense to me, but I'd appreciate it if we did this _after_ all the open kvutils PRs have been merged or closed, because it's gonna cause merge conflict hell.

Of course!

File: ledger/api-server-damlonx/reference-v2/BUILD.bazel (hunk 2 above)
Comment (miklos-da) [reply ledger/api-server-damlonx/reference-v2/BUILD.bazel]: Good point -- will do that.

**Comment by SamirTalwar** [comment]
If we're gonna rename this, can we rename it to something more meaningful? Perhaps "kv-participant" or "kv-ledger" or something?

**Comment by miklos-da** [comment]
> If we're gonna rename this, can we rename it to something more meaningful? Perhaps "kv-participant" or "kv-ledger" or something?

Let's discuss offline what the naming should be.

**Pull request closed without merging**